app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  # def create
+  #   @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+
+  #   if @user.save
+  #     redirect_to new_user_path
+  #   else
+  #     render :new
+  #   end
+  # end
```

Comment:
Remove commented code.

Reply:
Thanks for the review.
It is possible to edit the user in <yourlocalhost>/user/<id>/edit